lib/fog/kubevirt/compute/models/storageclass.rb

```diff
@@ -15,17 +18,21 @@ class Storageclass < Fog::Model
 
         def self.parse(object)
           metadata = object[:metadata]
-          {
+          annotations = metadata[:annotations]
+          sc = {
             :name                => metadata[:name],
             :resource_version    => metadata[:resourceVersion],
             :uid                 => metadata[:uid],
-
+            :default             => false,
```

Comment:
If the SC is the only one, and it has no annotation, wouldn't it mean that it is the actual default and this value might be misleading ? Perhaps we're better only with line 34.

Reply:
Will remove line 26. In my deployment one has the annotation. Maybe it is difference between the versions of the cluster or lack of setting the `default` one.